tools/clean-emoji-cache

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env python
+from __future__ import absolute_import
+from __future__ import print_function
+import os
+import subprocess
+import sys
+import hashlib
+
+ZULIP_PATH = os.path.dirname(os.path.dirname(os.path.abspath(__file__)))
+sys.path.append(ZULIP_PATH)
+from scripts.lib.emoji_utils import get_success_stamp
+
+EMOJI_SCRIPT_DIR_PATH = os.path.join(ZULIP_PATH, 'tools', 'setup', 'emoji')
+EMOJI_CACHE_PATH = "/srv/zulip-emoji-cache"
+if "--travis" in sys.argv:
+    EMOJI_CACHE_PATH = os.path.join(os.environ["HOME"], "zulip-emoji-cache")
+
+success_stamp = get_success_stamp(EMOJI_CACHE_PATH, EMOJI_SCRIPT_DIR_PATH)
```

Comment:
Taking a closer look, that symlink is generated after we calculate the hash in `get_success_stamp`.